test/index.test.js

```diff
@@ -0,0 +1,43 @@
+jest.mock('../ffc-pay-file-consumer/config', () => ({ totalRetries: 1 }))
+const consumer = require('../ffc-pay-file-consumer')
+const mockContext = require('./mock-context')
+jest.mock('../ffc-pay-file-consumer/storage')
+const mockStorage = require('../ffc-pay-file-consumer/storage')
+let message
+const file = {}
+const content = 'content'
+
+describe('consumer', () => {
```

Comment:
jest.mock('../ffc-pay-file-consumer/config', () => ({ totalRetries: 1 }))
jest.mock('../ffc-pay-file-sender/storage')
const mockStorage = require('../ffc-pay-file-sender/storage')
const mockContext = require('./mock-context')

const consumer = require('../ffc-pay-file-consumer')

const blob = {}
const content = 'content' // suggest a comment to define that this is merely the console/I assume whatever runtime env its in?

let message

describe('sender', () => {

Reply:
`content` just represents the content of the file rather than anything runtime related.